eng/config/PublishData.json

```diff
@@ -131,8 +132,8 @@
       ],
       "vsBranch": "main",
       "vsMajorVersion": 17,
-      "insertionCreateDraftPR": false,
-      "insertionTitlePrefix": "[d17.4p2]"
+      "insertionCreateDraftPR": true,
+      "insertionTitlePrefix": "[d17.4p3]"
```

Comment:
Should we add main-vs-deps back as well?

Reply:
Yes, but I would prefer to do that separately once we've snapped